src/client/Microsoft.Identity.Client/ApiConfig/GetAuthorizationRequestUrlParameterBuilder.cs

```diff
@@ -97,6 +101,27 @@ public GetAuthorizationRequestUrlParameterBuilder WithPkce(out string codeVerifi
             return this;
         }
 
+        /// <summary>
+        /// To help with resiliency, AAD Cached Credential Service (CCS) operates as an AAD backup.
```

Comment:
Not sure, maybe would be helpful to add a remark here that this method take precedence even if `WithLoginHint` is specified?